cypress/integration/tests/developer/applicationinventory/analysis/binary_analysis.test.ts

```diff
@@ -51,7 +51,7 @@ describe("Binary Analysis", { tags: "@tier1" }, () => {
         Proxy.disableAllProxies();
 
         // Enable HTML anc CSV report downloading
-        let reportConfig = ReportConfig.getInstance();
+        let reportConfig = GeneralConfig.getInstance();
```

Comment:
Please change "reportConfig" as well at all places.

Reply:
Done, PR is updated